cmd/edenStop.go

```diff
@@ -58,10 +58,18 @@ var stopCmd = &cobra.Command{
 		if eveRemote {
 			return
 		}
-		if err := eden.StopEVEQemu(evePidFile); err != nil {
-			log.Infof("cannot stop EVE: %s", err)
+		if eveParallels {
```

Comment:
The same: https://github.com/lf-edge/eden/blob/97877c68d13df9218f6ee798f55eb4aa9f2f742b/cmd/edenSetup.go#L159